frontend/test/e2e/support/utils.js

```diff
@@ -224,13 +227,32 @@ function jasmineMultipleSetupTeardown(fn) {
         const handlers = { beforeAll: [], beforeEach: [], afterEach: [], afterAll: [] }
         const originals = {};
 
+        let originalDescribe = global.describe;
```

Comment:
This all looks fine but I'm not entirely sure what it's purpose is. Might be worth dropping a note as to its reason for being.